server/src/main/java/org/elasticsearch/action/search/SearchResponse.java

```diff
@@ -985,10 +1025,12 @@ public static Cluster fromXContent(String clusterAlias, XContentParser parser) t
             Integer skippedShardsFinal = skippedShards == -1 ? null : skippedShards;
             Integer failedShardsFinal = failedShards == -1 ? null : failedShards;
             TimeValue tookTimeValue = took == -1L ? null : new TimeValue(took);
+            boolean skipUnavailable = false;  // skipUnavailable is not exposed to XContent, so just default to false here
```

Comment:
This is needed for tracking final state of a "failed" search on a cluster - is it "failed" or "skipped"?